src/main/java/com/capgemini/molveno/controller/api/TableController.java

```diff
@@ -0,0 +1,42 @@
+package com.capgemini.molveno.controller.api;
+
+import com.capgemini.molveno.model.Table;
+import com.capgemini.molveno.service.TableService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("api/tables")
+public class TableController {
+    @Autowired
+    private TableService tableService;
+
+    @PostMapping
```

Comment:
Doe dit bij de mappings, dit zorgt ervoor dat we later geen issues gaan krijgen omdat het een string is ipv een json object.

```
produces = MediaType.APPLICATION_JSON_VALUE
```